src/UniversalRewardsDistributor.sol

```diff
@@ -122,12 +122,16 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
             ErrorsLib.INVALID_PROOF_OR_EXPIRED
         );
 
+        require(claimable >= claimed[account][reward], ErrorsLib.ROOT_MISSCONFIGURED);
+
```

Comment:
This check feels very redundant with the check 2 lines below. Adding a `require(claimable > claimed[account][reward])` here and removing the check 2 lines below should be plenty already if we want to avoid reverting on underflows

Reply:
They are throwing for 2 different reasons. Maybe it's better to change the check of the line 129:

```diff
-         require(amount > 0, ErrorsLib.ALREADY_CLAIMED);
+         require(amount != 0, ErrorsLib.ALREADY_CLAIMED);

Reply:
Yes I think we should do this check `require(amount != 0, ErrorsLib.ALREADY_CLAIMED);` instead

Reply:
> They are throwing for 2 different reasons. 

Yes I know, but one of them, the `ROOT_MISSCONFIGURED` error, is thrown when the admins did a mistake and this is discovered in production, while potentially part of the rewards have already been distributed. This seems like an edge case, and I'm fine to put that together with the case that you are trying to claim too much (especially because it can be done nicely in the code)

Reply:
> > They are throwing for 2 different reasons.
> 
> Yes I know, but one of them, the `ROOT_MISSCONFIGURED` error, is thrown when the admins did a mistake and this is discovered in production, while potentially part of the rewards have already been distributed. This seems like an edge case, and I'm fine to put that together with the case that you are trying to claim too much (especially because it can be done nicely in the code)

I had the same opinion in the issue reported by Cantina in fact (see [here](https://github.com/cantinasec/review-morpho-blue-1/pull/67/files#r1366550666)). To me, it is less explicit if we don't throw an error and let underflow, but its ok since this is an offchain issue (Merkle tree misconfigured).